pkg/tracer/tracer_legacy.go

```diff
@@ -14,24 +14,20 @@ func (m *FlowFetcher) legacyLookupAndDeleteMap(met *metrics.Metrics) map[ebpf.Bp
 	iterator := flowMap.Iterate()
 	var flows = make(map[ebpf.BpfFlowId]model.BpfFlowContent, m.cacheMaxSize)
 	var id ebpf.BpfFlowId
-	var metrics []ebpf.BpfFlowMetrics
+	var baseMetrics ebpf.BpfFlowMetrics
 	count := 0
 
 	// Deleting while iterating is really bad for performance (like, really!) as it causes seeing multiple times the same key
 	// This is solved in >=4.20 kernels with LookupAndDelete
-	for iterator.Next(&id, &metrics) {
+	for iterator.Next(&id, &baseMetrics) {
 		count++
 		if err := flowMap.Delete(id); err != nil {
 			log.WithError(err).WithField("flowId", id).Warnf("couldn't delete flow entry")
 			met.Errors.WithErrorName("flow-fetcher-legacy", "CannotDeleteFlows").Inc()
 		}
 		// We observed that eBFP PerCPU map might insert multiple times the same key in the map
```

Comment:
nit: baseMetrics is no longer perCPU so this outdated comment

Reply:
done